docs/source/operations/projections/bertin1953.rst

```diff
@@ -0,0 +1,64 @@
+.. _bertin1953:
+
+********************************************************************************
+Bertin 1953
+********************************************************************************
+
+.. versionadded:: 5.2.0
+
++---------------------+-------------------------------+
+| **Classification**  |                               |
++---------------------+-------------------------------+
+| **Available forms** | Forward, spherical projection |
++---------------------+-------------------------------+
+| **Defined area**    | Global                        |
++---------------------+-------------------------------+
+| **Alias**           | bertin1953                    |
++---------------------+-------------------------------+
+| **Domain**          | 2D                            |
++---------------------+-------------------------------+
+| **Input type**      | Geodetic coordinates          |
++---------------------+-------------------------------+
+| **Output type**     | Projected coordinates         |
++---------------------+-------------------------------+
+
+
+
+.. figure:: ./images/bertin1953.png
+   :width: 500 px
+   :align: center
+   :alt:   Bertin 1953
+
+   proj-string: ``+proj=bertin1953``
+
+
+The Bertin 1953 projection is intended for making world maps. Created by Jacques Bertin
+in 1953, this projection was the go-to choice of the French cartographic school when they
+wished to represent phenomena on a global scale. The projection was formulated in 2017
+by Philippe Rivière for visionscarto.net.
+
+
+Usage
+###############################################################################
+
+The Bertin 1953 projection has no special options. Its rotation parameters
+are fixed. Here is an example of a forward projection with scale 1:
+
+    $ echo 122 47 | src/proj +proj=bertin1953 +R=1
+    0.72	0.73
+
+Parameters
+################################################################################
+
+.. note:: All parameters for the projection are optional.
+
+.. include:: ../options/R.rst
+
+.. include:: ../options/x_0.rst
+
+.. include:: ../options/y_0.rst
+
+Further reading
+################################################################################
+
+#. Philippe Rivière (2017). `Bertin Projection (1953) <https://visionscarto.net/bertin-projection-1953>`, Visionscarto.net.
```

Comment:
Is this the best reference we have? If possible, I'd like to have a reference to the mathematical description as well, preferably an academic paper.

Reply:
Well if you read that modest blog post, you'll know :)
(I wish I had time and resources to publish this as an academic paper.)

Reply:
I did, but it wasn't clear to me if something else was published.

You are welcome to use the PROJ docs as a platform for describing the mathematics behind the projection.

Reply:
PS: I agree that my construction is totally ad-hoc, and that another formula might be much nicer mathematically (esp. with second and third derivatives). It's the best I could come up with as a first approximation, and I'm very open to making an upgraded formula at some point (provided it still respects the original drawing).